### ISSUE TITLE
[Issue]: Subtitles with dubplicate timestamps are ignored

### ISSUE THREAD

**Issue by ERivierePEReN** [jellyfin/jellyfin-web#5181]
**Describe The Bug**
Some multi-lines subtitles are separated with different entries that have similar timestamps like that:
```
3
00:00:09,090 --> 00:00:12,030
- A huge reservoir of magic

4
00:00:09,090 --> 00:00:12,030
hidden underneath the castle?
```
With the current implementation, Jellyfin-web will only print the first subtitle line `- A huge reservoir of magic` and skip the other one(s).

**Expected Behavior**
Subtitles with similar timestamps should all be shown on the screen. I think the expected behavior from my previous example would be
```
- A huge reservoir of magic
hidden underneath the castle?
```
to shown on the screen.

**System (please complete the following information):**
 - Jellyfin Version: [e.g. 10.8.13]

**Additional Context**
I've already fixed the issue in local. If you think this behavior is indeed problematic, I could create an MR with my solution.

**Comment by thornbill** [comment]
Yes. Please open a PR so we can review.

**Comment by ERivierePEReN** [comment]
Well, silly me. Unless I've got myself mixed up while building the locally the project; I think that issue already got fixed in 10.9.0.
